NewDawn/NewDawn/ProfileViewController/ProfileGNBViewController.swift

```diff
@@ -57,6 +57,7 @@ class ProfileGNBViewController: UIViewController {
         polishTextField(textField: birthdayTextField)
         loadStoredFields()
         showDatePicker()
+        navigationController?.setNavigationBarHidden(true, animated: true)
```

Comment:
Can you make sure this will work if you register a new account?

Reply:
Since in registration flow itself, there's no navigationController around

Reply:
Yea it works.